src/lib/fy-dump.c

```diff
@@ -293,8 +293,7 @@ void fyp_debug_dump_input(struct fy_parser *fyp, const struct fy_input_cfg *fyic
 		break;
 	case fyit_stream:
 		fyp_scan_debug(fyp, "%s: stream=\"%s\" fileno=%d chunk=%zu\n", banner,
-				fyic->stream.name, fileno(fyic->stream.fp),
```

Comment:
The chunk is still present in the printf string; please remove and update the PR.

Reply:
Indeed. It was already removed in a subsequent commit.